source/_integrations/homekit.markdown

```diff
@@ -458,6 +458,25 @@ automation:
     packet: XXXXXXXX
 ```
 
+## Events
+
+This integration emits events as `homekit_state_change` on the event bus. This can be used for example in automations to known whether an entity's state got changed from HomeKit specifically. 
```

Comment:
```suggestion
The HomeKit integration emits events as `homekit_state_change` on the event bus. This can be used in automations to know whether an entity’s state was changed from HomeKit.
```